### ISSUE TITLE
waiting for new updates

### ISSUE THREAD

**Issue by daarkwulf** [xfarrow/locatemydevice#15]
some issues i found
i have one plus nord ce3 lite 5g phone
with oxygen 14 with all latest updates

what i face is that still after first sms it just turn on location but not sending the location untill the app is again open by us in mobile
but thats a good point that after first msg it get turn on location even its turned off. as per the adb grant permission

but still on google find my device its not giving locaation even if gps is on untill mobile have internet access
so i want u to plz give thaat feacture to make the app turn on internet and wifi and location together
which will actually make easy for us to get location even by google find device in emergency

**Comment by ferrum** [comment]
Hi, thank you for using this application and I am sorry you're experiencing issues with it. It does seem like there are some problems with the service being killed by Android.

I will investigate and try to release an update as soon as possible. Unfortunately I won't be able to dedicate too much time daily, but I'll try my best. Thank you for your patience, especially for such a delicate time-critical scenarios where a smartphone could be lost.

Best regards.

**Comment by daarkwulf** [comment]
Yes but also when I try to get cellinfo it's giving me the tac instead of
lac which has to use on opencellid..

Also help me in that plz
